whitelist.py

```diff
@@ -0,0 +1,39 @@
+# Because vulture only performs static analysis, this file is used
+# by vulture as a whitelist module and passed as a parameter when
+# running vulture so that it'll know that certain methods are being
+# used.
+
```

Comment:
That's irritating. Looking at this, I'm not sure vulture is actually worthwhile anymore -- still, you've already done the work, so we might as well move forward on it.